### PR TITLE
Submit symptoms bridge

### PR DESCRIPTION
**Pull request by i-schuetz**
The view models send the inputs to the Rust core now.

### CODE REVIEW

File: CoEpi/repo/SymptomRepo.swift [hunk 1]
```diff
@@ -5,46 +5,32 @@ import os.log
...
-        Symptom(id: "3", name: L10n.Symptom.dryCough),
-        Symptom(id: "4", name: L10n.Symptom.muscleAches),
-        Symptom(id: "5", name: L10n.Symptom.nasalCongestion),
-        Symptom(id: "6", name: L10n.Symptom.runnyNose),
-        Symptom(id: "7", name: L10n.Symptom.soreThroat),
-        Symptom(id: "8", name: L10n.Symptom.diarrhea),
-        Symptom(id: "9", name: L10n.Symptom.difficultyBreathing),
-        Symptom(id: "10", name: L10n.Symptom.lossOfSmellTaste),
-        Symptom(id: "11", name: L10n.Symptom.none)
+        Symptom(id: .fever, name: L10n.Symptom.fever),
```
Comment (duskoo) [inline CoEpi/repo/SymptomRepo.swift:19]: tiredness, runny nose, sore throat have been dropped?

Comment (i-schuetz) [reply CoEpi/repo/SymptomRepo.swift:19]: Hmm I oriented with Android's symptom ids, which I got from Figma IIRC. I'll ask.

File: CoEpi/native/LibResult.swift [hunk 2]
```diff
@@ -42,6 +43,9 @@ extension Unmanaged where Instance == CFString {
         let resultValue: CFString = takeRetainedValue()
         let resultString = resultValue as String
 
+        os_log("Deserializing native core result: %{public}@, body: %{public}@", log: servicesLog,
```
Comment (duskoo) [inline CoEpi/native/LibResult.swift:46]: @i-schuetz I got  a crash here. Will submit PR.

Comment (duskoo) [reply CoEpi/native/LibResult.swift:46]: #188

**Pull request merged**